mhs/operating-mhs-adaptor.md

```diff
@@ -50,21 +50,24 @@ Log messages produced by the MHS Adaptor are in human readable form and importan
 and used for searching, filtering, agregation, graphing etc.
```

Comment:
Might be good to mention that few first logs don't follow the pattern as they are written before logger initialization. Those are minor read-config logs